cypress/e2e/orangeHRM/Scenario1.cy.ts

```diff
@@ -0,0 +1,89 @@
+import AddEmployeePage from "cypress/support/pageObjects/mainPages/AddEmployeePage";
+import AddEntitlementsPage from "cypress/support/pageObjects/mainPages/Leave/Entitlements/AddEntitlementsPage";
+import ApplyLeavePage from "cypress/support/pageObjects/mainPages/Leave/ApplyLeavePage";
+import LoginPage from "cypress/support/pageObjects/mainPages/LoginPage";
+import LeaveListPage from "cypress/support/pageObjects/mainPages/Leave/LeaveListPage";
+import Sidebar from "cypress/support/pageObjects/subPages/Sidebar";
+import MyLeavePage from "cypress/support/pageObjects/mainPages/Leave/MyLeavePage";
+import table from "cypress/support/pageObjects/objects/table";
+
+const myLoginPage: LoginPage = new LoginPage();
+const myAddEmployeePage: AddEmployeePage = new AddEmployeePage()
+const myAddEntitlementsPage: AddEntitlementsPage = new AddEntitlementsPage();
+const myApplyLeavePage: ApplyLeavePage = new ApplyLeavePage();
+const myLeaveListPage: LeaveListPage = new LeaveListPage();
+const mySidebar: Sidebar = new Sidebar();
+const my_MyLeavePage: MyLeavePage = new MyLeavePage();
+describe('Scenario1', () => {
+
+    before('Scenario1 (Given)',() => {
+        cy.log('***************Create New Employee***************')
+
+        // visiting orangeHRM website
+        cy.visit("/");
+        
+        // login
+        myLoginPage.login("Admin", "admin123");
+
+        // declaring a fixture
+        cy.fixture('employeeInfo').as('empInfo');
+
+        // Create Employee
+        cy.get('@empInfo').then((empData: any) => {
+            // Given The system has an Employee with Login Details
+            myAddEmployeePage.addWithLoginViaAPI(empData).then((body: any) => {
+                empData.empNumber = body.data.employee.empNumber;
```

Comment:
you can modify your API to return the data directly instead of reading it here, so you can get rid of "body.data"